src/ballet/sbpf/fd_sbpf_loader.c

```diff
@@ -428,7 +428,10 @@ fd_sbpf_load_shdrs( fd_sbpf_elf_info_t *  info,
            separately done in the sBPF verifier. */
 
   info->text_off = (uint)shdr_text->sh_offset;
-  info->text_cnt = (uint)shdr_get_loaded_size( shdr_text ) / 8U;
+  ulong text_size = shdr_get_loaded_size( shdr_text );
```

Comment:
Can this be unaligned?

Reply:
Alignment checks are done later on in `fd_vm_validate`

Reply:
Agave allows text section size to be unaligned during ELF loading

Reply:
Thanks. Scary! :)